x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_visualization.tsx

```diff
@@ -18,25 +18,24 @@ import { generateId } from '../id_generator';
 export const xyVisualization: Visualization<State, PersistableState> = {
   getSuggestions,
 
-  initialize(datasource, state) {
+  initialize(frame, state) {
     return (
       state || {
-        seriesType: 'bar',
         title: 'Empty XY Chart',
         legend: { isVisible: true, position: Position.Right },
-        x: {
-          accessor: generateId(),
-          position: Position.Bottom,
-          showGridlines: false,
-          title: 'X',
-        },
-        y: {
-          accessors: [generateId()],
-          position: Position.Left,
-          showGridlines: false,
-          title: 'Y',
-        },
-        splitSeriesAccessors: [generateId()],
+        layers: [
+          {
+            layerId: frame.addNewLayer(),
+            accessors: [generateId()],
+            datasourceId: '',
+            position: Position.Top,
```

Comment:
Position.TOP seems like a weird default

Reply:
Agreed. It seems to be being ignored, too... So, it's always on the right currently.

Reply:
Actually... what is this position supposed to be used for? It doesn't seem to be doing anything at the moment.

Reply:
I'm going to redo this in a future PR- it's for axis positioning